.buildkite/pipeline.testing.yml

```diff
@@ -9,4 +9,4 @@ env:
 steps:
   - label: ":aws: E2E tests in AWS"
     command:
-      - echo "TODO: make this work again"
+      - "echo 'TODO: make this work again'"
```

Comment:
According to yamllint you NEED the outside quotes otherwise yaml interprets it as  a key value block , ie
"echo 'TODO" = "make this work again'"